Aptoide/aptoide-client-v8
# [task/ASV-1410] Remove Google Login in Cobrand

**What does this PR do?**

   Removes Google Login for cobrand.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] LoginPreferences.java

**How should this be manually tested?**

  Check if there's no Google login for cobrand but also check if vanilla google login still works. 

**What are the relevant tickets?**

  [ASV-1410](https://aptoide.atlassian.net/browse/ASV-1410)


**Code Review Checklist**
